python/sdist/setup_clibs.py

```diff
@@ -218,7 +220,10 @@ def getLibAmici(extra_compiler_flags=None, h5pkgcfg=None, blaspkgcfg=None):
                          'amici/ThirdParty/sundials/src',
                          'amici/ThirdParty/gsl/',
                          ],
-        'cflags': ['-std=c++11', *extra_compiler_flags],
+        'cflags': [*extra_compiler_flags],
+        'cflags_mingw32': ['-std=c++11'],
+        'cflags_unix': ['-std=c++11'],
+        'cflags_msvc': ['/std:c++17'],
```

Comment:
Do we need 17 here? may (unnecessarily?) exclude older msvc versions here.

Reply:
The compiler msvc started using this option with version 14, so the possibilities we have are 
`/std:[c++14|c++17|c++latest]`. I am pretty sure that all 3 work for AMICI, but I might be overlooking something.

Reply:
if all work with amici, i would go for the lowest one that works

Reply:
I have changed these occurrences of `-std=c++11` and `/std:c++17` to `-std=c++14` and `/std:c++14`. As far as I can tell by searching, that is all of them.